doc/ref_model/chapters/chapter03.md

```diff
@@ -285,7 +285,29 @@ The terms Container Infrastructure Service Instance and Container Infrastructure
 ### 3.8.1 Types of accelerators
 
 <a name="3.8.2"></a>
-### 3.8.2 Levels of accelerator consumption
+### 3.8.2 Infrastructure and Application Level Acceleration
+
+Figure 3-8 gives examples for Hardware Accelerators in [Sample reference model realization](#3.7) diagram.
+
+<p align="center"><img src="../figures/ch03-hardware-acceleration-in-rm-realization-diagram.png" alt="Hardware Acceleration in RM Realization Diagram" Title="Hardware Acceleration in RM Realization Diagram" width="65%"/></p>
+<p align="center"><b>Figure 3-6:</b> Hardware Acceleration in RM Realization Diagram</p>
+
+Hardware Accelerators are part of the Hardware Infrastructure Layer. Those that need to be activated/programmed will expose management interfaces and have Accelerator Management software reaching them in-band (from host OS) or out of band (OOB, over some network to the adapter without going through host OS). For more flexibility in management, such Accelerator Management can be carried over appropriate service with authentication mechanism before being exposed to Cloud Infrastructure operator and/or Application tenant.
```

Comment:
"software reaching them in-band" ---> "software managing them in-band ..."

Reply:
> 
> 
> "software reaching them in-band" ---> "software managing them in-band ..."

Changed. Thanks